tests/contrib/test_sklearn.py

```diff
@@ -76,6 +90,27 @@ def test_hook(model_fixture, inp_data, request):
     assert signature.returns == returns
 
 
+def test_hook_transformer(transformer, inp_data_text):
+    data_type = DataAnalyzer.analyze(inp_data_text)
+    model_type = ModelAnalyzer.analyze(transformer, sample_data=inp_data_text)
+    assert isinstance(model_type, SklearnTransformer)
+    assert TRANSFORM_METHOD_NAME in model_type.methods
+    signature = model_type.methods[TRANSFORM_METHOD_NAME]
+    returns = ScipySparceMatrix(dtype="float64")
+    assert signature.name == TRANSFORM_METHOD_NAME
+    assert signature.args[0] == Argument(name="raw_documents", type_=data_type)
+    assert signature.returns == returns
+
+
+def test_model_type__transform(transformer, inp_data_text):
+    model_type = ModelAnalyzer.analyze(transformer, sample_data=inp_data_text)
+
+    np.testing.assert_array_almost_equal(
+        transformer.transform(inp_data_text).todense(),
+        model_type.call_method("transform", inp_data_text).todense(),
+    )
+
+
```

Comment:
can you add a test for using transformer as a preprocessor? It's a new feature in 0.4.0 and it could be quite useful for transformers

Reply:
and also tests for scipy ext

Reply:
@ykasimov you can use this as an example for what Mike suggests https://github.com/aguschin/example-get-started/pull/1/files

Reply:
I added the test but it's not working with OneHotEncoder. it seems that after I save the model and then load it, the preprocessing isn't applied, what could be the issue here?

Reply:
fixed. changed `load` to `load_meta`